gcloud/_helpers.py

```diff
@@ -439,8 +441,7 @@ def _to_bytes(value, encoding='ascii'):
     :rtype: str / bytes
     :returns: The original value converted to bytes (if unicode) or as passed
               in if it started out as bytes.
-    :raises: :class:`TypeError <exceptions.TypeError>` if the value
-             could not be converted to bytes.
+    :raises TypeError: if the value could not be converted to bytes.
```

Comment:
@tseaver This is what `pylint` requires to "believe" that `TypeError` is mentioned in the `Raises` section. Unfortunately, the Sphinx generated docs never turn this into a link (not a big deal for stdlib exceptions, but it's nice to have links for custom exceptions)